static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -143,6 +144,18 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
             replayId={replayId}
           />
         )}
+        {eventData && (
+          <Section icon={<IconIssues size="xs" />} title={t('Issue Tracking')}>
+            <ErrorBoundary mini>
+              <ExternalIssueList
+                project={feedbackItem.project}
+                group={feedbackItem as unknown as Group}
```

Comment:
typescript was unhappy casting the `feedbackItem` as `Group` so i had to add `as unknown` here